lizmap/modules/admin_api/lib/RepoCreator.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace LizmapApi;
+
+class RepoCreator
+{
+    /**
+     * Creates a new repository with the given properties.
+     *
+     * @param string $key                    The unique identifier for the repository. Must not yet exist.
+     * @param string $label                  the display name of the repository
+     * @param string $path                   the path to the repository directory
+     * @param string $allowUserDefinedThemes whether user-defined themes are allowed for the repository
```

Comment:
```suggestion
     * @param bool $allowUserDefinedThemes whether user-defined themes are allowed for the repository
```
It has to be a boolean, isn't it